### ISSUE TITLE
v0.3.0 windows build fails to run due to libMonoPosixHelper being bad image format

### ISSUE THREAD

**Issue by Licho1** [microsoft/sbom-tool#179]
```
sbom-tool-win-x64 (1)\mKUTLbqWlQzcwLm3HUQsFUT02rtVQ8c=\libMonoPosixHelper.dll'
   at System.Reflection.AssemblyName.nGetFileInformation(String s)
   at System.Reflection.AssemblyName.GetAssemblyName(String assemblyFile)
   at Microsoft.Sbom.Api.Bindings.<>c.<Load>b__0_8[T](String x)
```

**Comment by edgarrs** [comment]
Hi @Licho1, can you please share detailed repro steps so we can try it on our side?

**Comment by Rtalos** [comment]
I am also experiencing this.

- Azure pipeline yaml
- Self-hosted windows agent

**Failing scenario with Powershell task and error output:**
```yaml
  - task: PowerShell@2
    displayName: Generate SBOM
    inputs:
      targetType: 'inline'
      script: |
        Invoke-WebRequest -Uri "https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-win-x64.exe" -OutFile "$(Agent.TempDirectory)\sbom-tool.exe"
        
        $(Agent.TempDirectory)\sbom-tool generate -b $(Build.ArtifactStagingDirectory) -bc $(Build.SourcesDirectory) -pn Test -pv 1.0.0 -ps ${{parameters.companyName}} -nsb ${{parameters.websiteUrl}} -V Verbose
```

![image](https://user-images.githubusercontent.com/19684631/204848622-cf48ffdd-3a6a-4d23-8dfa-623272f09d84.png)


**Working scenario with Powershell task:**
```yaml
  - task: PowerShell@2
    displayName: Generate SBOM
    inputs:
      targetType: 'inline'
      script: |
        Invoke-WebRequest -Uri "https://github.com/microsoft/sbom-tool/releases/download/v0.2.7/sbom-tool-win-x64.exe" -OutFile "$(Agent.TempDirectory)\sbom-tool.exe"
        
        $(Agent.TempDirectory)\sbom-tool generate -b $(Build.ArtifactStagingDirectory) -bc $(Build.SourcesDirectory) -pn Test -pv 1.0.0 -ps ${{parameters.companyName}} -nsb ${{parameters.websiteUrl}} -V Verbose
```

**Full stack trace if needed:**
```
Unhandled exception. System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.BadImageFormatException: 
File name: 'C:\Windows\ServiceProfiles\NetworkService\AppData\Local\Temp\.net\sbom-tool\mKUTLbqWlQzcwLm3HUQsFUT02rtVQ8c=\libMonoPosixHelper.dll'
   at System.Reflection.AssemblyName.nGetFileInformation(String s)
   at System.Reflection.AssemblyName.GetAssemblyName(String assemblyFile)
   at Microsoft.Sbom.Api.Bindings.<>c.<Load>b__0_8[T](String x)
   at System.Linq.Enumerable.SelectArrayIterator`2.ToArray()
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at Microsoft.Sbom.Api.Bindings.<>c__DisplayClass0_0.<Load>g__BindAllTypesThatImplement|2[T](String[] partialAssemblyNames)
   at Microsoft.Sbom.Api.Bindings.Load()
   at Ninject.Modules.NinjectModule.OnLoad(IKernel kernel)
   at Ninject.KernelBase.Load(IEnumerable`1 m)
   at Ninject.KernelBase..ctor(IComponentContainer components, INinjectSettings settings, INinjectModule[] modules)
   at Ninject.KernelBase..ctor(INinjectModule[] modules)
   at Ninject.StandardKernel..ctor(INinjectModule[] modules)
   at Microsoft.Sbom.Api.Config.SbomToolCmdRunner..ctor()
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
   --- End of inner exception stack trace ---
   at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic, Boolean wrapExceptions)
   at System.Activator.CreateInstance(Type type)
   at PowerArgs.ObjectFactory.CreateInstance(Type t)
   at PowerArgs.Args.ParseInternal(CommandLineArgumentsDefinition definition, String[] input)
   at PowerArgs.Args.<>c__DisplayClass17_1.<InvokeAction>b__3()
   at PowerArgs.Args.Execute[T](Func`1 argsProcessingCode)
   at PowerArgs.Args.<>c__DisplayClass17_0.<InvokeAction>b__2(String[] a)
   at PowerArgs.REPL.DriveREPL[T](TabCompletion t, Func`2 eval, String[] args, Func`1 defaultEval)
   at PowerArgs.Args.InvokeAction(CommandLineArgumentsDefinition definition, String[] args)
   at PowerArgs.Args.InvokeAction[T](String[] args)
   at PowerArgs.Args.<>c__DisplayClass32_0`1.<InvokeActionAsync>b__0()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__272_0(Object obj)
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop(Thread threadPoolThread, ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location ---
   at Microsoft.Sbom.Tool.Program.Main(String[] args)
   at Microsoft.Sbom.Tool.Program.<Main>(String[] args)
```

**Comment by aasim** [comment]
This issue has been fixed in version 0.3.1
